expression/builtin_string_vec.go

```diff
@@ -17,6 +17,7 @@ import (
 	"encoding/base64"
 	"encoding/hex"
 	"fmt"
+	"github.com/pingcap/tidb/sessionctx"
```

Comment:
Please format this.

Reply:
BTW there's an error when I git commit

```
Checking for files that need goword... .git/hooks/pre-commit: line 58: ./tools/bin/goword: No such file or directory
OK
```
 Should I install https://github.com/chzchzchz/goword ?